ScoutSuite/frontend/src/components/ServiceCard/index.js

```diff
@@ -1,8 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faChevronRight } from '@fortawesome/free-solid-svg-icons';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
```

Comment:
i removed all uses of FontAwesome icons, if it's ok with you, we'll be able to remove the library from our package.json

Reply:
Yep! Good with me